CLAUDE.md

```
@@ -171,6 +171,22 @@ docker-compose restart duckdb-frontend
 
 ## Architecture Overview
 
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)